website/docs/r/compute_instance_v2.html.markdown

```diff
@@ -317,8 +317,9 @@ The following arguments are supported:
     the server. Changing this creates a new server.
 
 * `network` - (Optional) An array of one or more networks to attach to the
-    instance. The network object structure is documented below. Changing this
-    creates a new server.
+    instance. Required when there are multiple networks difined for the tenant
```

Comment:
`Required when there are multiple networks defined for the tenant.` (fixed spelling on **defined** and added period at the end)

Reply:
done